src/requests.jl

```diff
@@ -70,10 +70,20 @@ end
     return URIs.resolvereference(base_url, _path)
 end
 
+const _common_docstring_request = """
+- `headers::AbstractDict = Dict{String, String}()`: headers of the request.
+- `query::Union{AbstractDict, Nothing} = nothing`: query parameters.
+- `require_base_url::Symbol = :strict`: to what extent the requested URL has to match the base URL of the `client`. Possible values are:
+    - `:strict` (requested URL has to start with the base URL),
+    - `:host` (host and scheme of the requested URL and base URL have to be equal),
+    - `:scheme` (scheme of the requested URL and base URL have to be equal),
+    - `:no` (requested URL does not have to match the base URL).
+"""
+
 """
     request_raw(
         client::Client, verb::AbstractString, path::AbstractString;
-        <keyword arguments>
+        <keyword arguments>...
```

Comment:
The current version is based on the Julia documentation which suggests using `<keyword arguments>` without `...`: https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation